client/src/components/plugins/icons.js

```diff
@@ -82,6 +82,32 @@ library.add(
     faWrench
 );
 
+const glFilterSolid = {
```

Comment:
You should probably be able to use the standard fontawesome regular and solid versions here, instead of a custom definition.

Reply:
Are you referring to the naming, or the icons themselves?

Reply:
The actual icons; we can use faFilter in `solid` or `regular` variants.

Reply:
https://fontawesome.com/icons/filter?s=regular

faFilter-regular is a pro icon. Do we have a Pro subscription?

Reply:
It's not a copy, but a custom icon I made for this PR

Reply:
The icons look great, but can custom icons not also cause issues when e.g. update font-awesome versions?